content/applications/finance/payment_acquirers/adyen.rst

```diff
@@ -58,15 +59,19 @@ this, log into your Adyen account then go to :menuselection:`Developers --> Webh
    :align: center
    :alt: Configure a webhook.
 
-There, in :menuselection:`Transport --> URL`, enter your server address followed by
-`/payment/adyen/notification`.
+There, in :menuselection:`General --> Server configuration --> URL`, enter your server address
+followed by `/payment/adyen/notification`.
 
-.. image:: adyen/adyen_webhook_url.png
+.. image:: adyen/adyen-webhook-url.png
    :align: center
    :alt: Enter the notification URL.
 
-Then continue in :menuselection:`Additional Settings --> HMAC Key --> Generate new HMAC key`. Be
-careful to copy it as you'll not be allowed to get it later without generating a new one.
+Then enter :menuselection:`Security --> HMAC Key --> Generate`. Be careful to copy the key as you
+will not be allowed to do it later without generating a new one.
+
+.. image:: adyen/adyen-hmac-key.png
```

Comment:
there are still the two versions of the image file hmac-key: delete the original and rename the converted one